### ISSUE TITLE
Lightcurve plotter not working complaining about missing WebGL support in broswer

### ISSUE THREAD

**Issue by kirxkirx** [snad-space/ztf-viewer#235]
The lightcurve plotter fails to make a plot on some objects (but not others) complaining about missing WebGL support in my browser.

URL of the object that fails to plot: https://ztf.snad.space/dr8/view/774116300006409 
(the search coordinates were `00:47:36.25 +57:27:02.9`)

Browser: Firefox 95.0.1 (64 bit)

More system info:
```
System:    Kernel: 5.11.8-gentoo-r1-x86_64 x86_64 (64 bit) Desktop: Fluxbox 1.3.7-gentoo-r4
           Distro: Gentoo Base System release 2.7
Machine:   System: FUJITSU product: LIFEBOOK E744 v: 10601583233
           Mobo: FUJITSU model: FJNB26F v: L3 Bios: FUJITSU // Phoenix v: Version 1.08 date: 03/10/2014
CPU:       Quad core Intel Core i7-4702MQ (-HT-MCP-) cache: 6144 KB 
           clock speeds: max: 3200 MHz 1: 800 MHz 2: 800 MHz 3: 1000 MHz 4: 1985 MHz 5: 900 MHz 6: 800 MHz
           7: 800 MHz 8: 900 MHz
Info:      Processes: 269 Uptime: 1 day Memory: 6235.1/7839.1MB Client: Shell (bash) inxi: 2.2.19
```
```
# lspci | grep VGA
00:02.0 VGA compatible controller: Intel Corporation 4th Gen Core Processor Integrated Graphics Controller (rev 06)
```

The behavior is affected by firefox `about:config` setting `webgl.min_capability_mode`: when changed to false, the misisng WebGL error message disappears and the plot is produced, but it seems to get rendered improperly (some colors are missing).

**Comment by hombit** [comment]
Thank you for the report! Plotly uses WebGL backend for plots having > 1000 points. We can disable it globally or conditionally (see https://get.webgl.org/ for an example of WebGL support detection) via `render_mode='svg'` ([Plotly docs for this](https://plotly.com/python/webgl-vs-svg/))

**Comment by hombit** [comment]
@kirxkirx  Please review a fix #256 at http://pr256.ztf.snad.space/dr13/view/774116300006409

**Comment by kirxkirx** [comment]
Yes, the fix is working! 

In the mean time I've upgraded to Firefox 107.0 and can report the problem was present with this version of the browser (link to the production version in the first post)
![Screenshot2022-11-25at10-04-01SNADZTFviewer](https://user-images.githubusercontent.com/42295873/203958684-da52cebc-5f1c-4200-ae81-f095930604ef.png)
and is fixed now (link to the test version)
![Screenshot2022-11-25at10-05-48SNADZTFviewer](https://user-images.githubusercontent.com/42295873/203959182-7b0e637e-0951-4e04-bbfd-d54a12d45473.png)
